routes/alumni.js

```diff
@@ -14,4 +15,15 @@ router.get('/', function(req, res, next) {
   })
 });
 
+router.get('/:id', function(req, res, next) {
+  alumni.findById(req.params.id, (err, alum) => {
+    if (err) {
+      console.error("couldnt get alumn", err)
+      res.send('couldnt get alumni');
```

Comment:
so this is going to send back a 200 response rather than an error, in this case you will want to send a 404